code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -36,6 +36,13 @@ All effects don't start immediately, but rather get worse over time; the rate is
 /datum/reagent/consumable/ethanol/on_mob_life(mob/living/M)
 	if(ishuman(M))
 		var/mob/living/carbon/human/H = M
+		var/obj/item/organ/alcoholvessel/dwarf
+		dwarf = H.getorganslot("dwarf_organ")
```

Comment:
The intent is to take a xenomorph plasma module approach, as demonstrated by the design document spec.